doc/rtt.js

```diff
@@ -209,11 +209,15 @@ export function start(serialnumber, startoptions, callback) {}
 export function stop(callback) {}
 
 /**
- * Async function to read RTT contents from an up channel on the device. You read on the up channel
+ * <p>Async function to read RTT contents from an up channel on the device. You read on the up channel
  * specified by the <tt>channelIndex</tt>. The callback function can access a UTF-8 string representation
  * of the RTT message (<tt>stringData</tt> in the example) if UTF-8 encoding is possible, as well
  * as an array (<tt>rawData</tt> in the example) containing the byte values (as integers between 0
- * and 255), plus the time elapsed since RTT was started, in microseconds.
+ * and 255), plus the time elapsed since RTT was started, in microseconds.</p>
+ *
+ * <p>This fumction will attepmt read up to <tt>length</tt> number of bytes and return them as both
```

Comment:
s/fumction/function/

Could we reflect that this call is not waiting for the requested amount of data? For me 'attempting' to read doesn't necessarily mean it's not waiting for all that data.
This function returns with the at most `length` bytes of content of the SEGGER buffer at the time of calling...

I leave it up to you, I'm not sure myself.